google_shopping.js

```diff
@@ -1,5 +1,59 @@
 var data = require("./products.json")
+var items = data.items;
+var counter = 0;
+var images = [];
+var canon = [];
+var eBayCanon = [];
 
 // Write your solutions below
 
-console.log(data["items"]);
\ No newline at end of file
+// console.log(data["items"]);
+
+for( var i = 0; i < items.length; i++){
+	if(items[i].kind == "shopping#product"){
+		counter++;
+		console.log('found product');
+	}
+}
+console.log(counter);
+
+for(var i =0; i < items.length; i++){
+	if(items[i].product.inventories[0].availability == "backorder"){
+		console.log(items[i].product);
+	}
+}
+
+
+for( var i = 0; i < items.length; i++){
+	var brand = items[i].product.brand;
+	var price = items[i].product.inventories[0].price;
+	var imageLink = items[i].product.images[0].link
+}
+
+for( var i = 0; i < items.length; i++){
```

Comment:
For this one, let's move the console log outside of the loop so that it only prints once, after the loop is finished. Also, a better practice to append items to an array is to use the .push() method, like so:

```for( var i = 0; i < items.length; i++){``
	```if(items[i].product.images.length > 1){```
		```images.push( items[i].product.title );```
	```};```
```}```
```console.log(images);```